File: tests/java/src/main/java/com/escape/tests/cases/Case177DeadBranchGlobal08.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 177: dead_branch_global_08 false-positive/false-negative stress pattern. */
public class Case177DeadBranchGlobal08 {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        String taskName = "dead_branch_global_08";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "stress");
        payload.put("stage", "evaluation");
        payload.put("input", raw);
        payload.put("checksum", taskName + ":" + raw.length());
        if (false) {
            RETAINED_CACHE.put("case_177", payload);
        }
        // SAFE: dead branch retention cannot execute.
        return payload.get("checksum");
    }
}

File: tests/go/cases/case_177_dead_branch_global_08.go
package escape_tests

var retainedCase177 = []map[string]string{}

func Case177DeadBranchGlobal08(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_branch_global_08",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "dead_branch_global_08:" + raw,
	}
	if false {
		retainedCase177 = append(retainedCase177, payload)
	}
	// SAFE: dead branch retention.
	return payload["checksum"]
}
